README.md

```diff
@@ -1,4 +1,4 @@
 # Developer calls agendas and transcripts
 * Jitsi link to join the call https://meet.jit.si/RGBcall1
-* Time of the call - 16:00 UTC
-* Discussions are recorded and uploaded online
+* Time of the call - 16:00 UTC, every second wednesday
+* Discussions are recorded and uploaded in this repository
```

Comment:
Good point. 
Although we are also planning to upload the recordings to YouTube with time, currently this repo is the only source.

Can you please change "uploaded in" -> "uploaded to".
Thanks!

Reply:
```suggestion
* Discussions are recorded and uploaded to this repository
```